frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-steps.ts

```diff
@@ -0,0 +1,114 @@
+import { TFunction } from 'i18next';
+import { WizardStep } from '@patternfly/react-core';
+import { WizardState } from '../reducer';
+import { BackingStorageType, RHCS, StepsId } from '../../../constants/create-storage-system';
+
+export const createSteps = (
+  t: TFunction,
+  state: WizardState,
+  hasStorageCluster: boolean,
+): WizardStep[] => {
+  const { backingStorage, currentStep } = state;
+  const { count } = currentStep;
+
+  const commonSteps = {
+    capacityAndNodes: {
+      id: StepsId.CapacityAndNodes,
+      name: t('ceph-storage-plugin~Capacity and nodes'),
+    },
+    securityAndNetwork: {
+      id: StepsId.SecurityAndNetwork,
+      name: t('ceph-storage-plugin~Security and network'),
+    },
+    reviewAndCreate: {
+      id: StepsId.ReviewAndCreate,
+      name: t('ceph-storage-plugin~Review and create'),
+    },
+  };
```

Comment:
Wizard steps are type of Nav
```suggestion
  const commonSteps = {
    capacityAndNodes: {
      id: StepsId.CapacityAndNodes,
      name: t('ceph-storage-plugin~Capacity and Nodes'),
    },
    securityAndNetwork: {
      id: StepsId.SecurityAndNetwork,
      name: t('ceph-storage-plugin~Security and Network'),
    },
    reviewAndCreate: {
      id: StepsId.ReviewAndCreate,
      name: t('ceph-storage-plugin~Review and Create'),
    },
  };
```
Wizard steps are type of navigation items (i presume) , They should be in title case ? http://openshift.github.io/openshift-origin-design/conventions/documentation/capitalization.html#title-case

Reply:
I think it's not true for wizards. But I can confirm once with ocp-console.

Reply:
Hey, an update. I checked with ocp console ux, its not true for wizards,
Wizards are supposed to use sentence case